src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -47,155 +52,188 @@ class Files(SdkApi):
         """Use uss.list() instead of this deprecated function."""
         return self.uss.list(path)
 
-    def get_file_content_streamed(self, file_path, binary=False):
+    def get_file_content_streamed(self, file_path: str, binary: bool =False)  -> Response:
         """Use uss.get_content_streamed() instead of this deprecated function."""
         return self.uss.get_content_streamed(file_path, binary)
 
-    def get_file_content(self, filepath_name):
+    def get_file_content(self, filepath_name: str) -> Optional[str]:
         """Use uss.get_content() instead of this deprecated function."""
         return self.uss.get_content(filepath_name)
 
-    def delete_uss(self, filepath_name, recursive=False):
+    def delete_uss(self, filepath_name: str, recursive: bool =False) -> None:
         """Use uss.delete() instead of this deprecated function."""
         return self.uss.delete(filepath_name, recursive)
 
-    def list_dsn(self, name_pattern, return_attributes=False):
+    def list_dsn(self, name_pattern: str, return_attributes: bool =False) -> DatasetListResponse:
         """Use ds.list() instead of this deprecated function."""
         return self.ds.list(name_pattern, return_attributes)
 
-    def list_dsn_members(self, dataset_name, member_pattern=None, member_start=None, limit=1000, attributes="member"):
+    def list_dsn_members(self,
+        dataset_name: str, 
+        member_pattern: Optional[str] = None, 
+        member_start: Optional[str] = None, 
+        limit: int = 1000, 
+        attributes: str ="member"
+    ) -> MemberListResponse:
         """Use ds.list_members() instead of this deprecated function."""
-        return self.ds.list_members(dataset_name, member_pattern, member_start, limit, attributes)
+        return self.ds.list_members(dataset_name, member_pattern, member_start, limit, attributes)        
 
     def copy_uss_to_data_set(
-        self, from_filename, to_dataset_name, to_member_name=None, type=FileType.TEXT, replace=False
-    ):
+        self, 
+        from_filename: str, 
+        to_dataset_name: str, 
+        to_member_name:Optional[str]=None, 
+        type: FileType=FileType.TEXT, 
+        replace: bool=False
+    ) -> None:
         """Use ds.copy_uss_to_data_set() instead of this deprecated function."""
         return self.ds.copy_uss_to_data_set(from_filename, to_dataset_name, to_member_name, type, replace)
 
     def copy_data_set_or_member(
         self,
-        from_dataset_name,
-        to_dataset_name,
-        from_member_name=None,
-        volser=None,
-        alias=None,
-        to_member_name=None,
-        enq=None,
-        replace=False,
-    ):
+        from_dataset_name: str,
+        to_dataset_name: str,
+        from_member_name: Optional[str] = None,
+        volser: Optional[str] = None,
+        alias: Optional[bool] = None,
+        to_member_name: Optional[str] = None,
+        enq: Optional[str] = None,
+        replace: bool = False,
```

Comment:
(nitpick) I noticed here we have spaces between the equals and the value up here, but down below we don't use spaces.

Not specifically requesting changes for this, but maybe running the formatter will resolve these formatting inconsistencies